source/class/qx/ui/treevirtual/SimpleTreeDataModel.js

```diff
@@ -795,7 +795,7 @@ qx.Class.define("qx.ui.treevirtual.SimpleTreeDataModel", {
      * Sets the whole data en bulk, or notifies the data model that node
      * modifications are complete.
      *
-     * @param nodeArr {Array | null}
+     * @param [nodeArr] {Array | null}
```

Comment:
Does the API Vieiwer understand this syntax? We've talked about updating what we allow inside of the curly braces, but not brackets around the variable name...

Reply:
The most common way, in qooxdoo code, of expressing this with an optional array that assumes a default value of null, would be:
```
@param nodeArr {Array?null}`
```

Does that stop your IDE from complaining?

Reply:
that's not correct syntax.  It should be:

```
@param nodeArr {Array?}
```

Reply:
Great, that works, thanks

Reply:
I think @johnspackman's comment about incorrect syntax was pertaining to what was in the PR, not my suggestion. Throughout qooxdoo, there are examples of what I suggested... but if you don't care that an optional parameter is replaced by null (you only care that it's optional), then John's syntax is more common.

Reply:
yes, that's right sorry for the confusion...